src/helper.js

```diff
@@ -188,6 +206,25 @@ function setHome(newPath) {
 	}
 }
 
+// Returns whether there was any change to the configuration.
```

Comment:
Can we split this up into `reloadConfig` and (as an example) `doesConfigHaveChanges`? I don't see why reloadConfig() should return a boolean of having changes.

Reply:
`doesConfigHaveChanges` would also have to `require()` the config, which feels wasteful

Reply:
the require would be in reloadConfig, doesConfigHaveChanges would call that, right? Just this way the diffing can be left for if it's needed.

Reply:
Are you saying `doesConfigHaveChanges()` should actually perform the reload? This is very unclear from the name.

Reply:
👍